log_outgoing_requests/models.py

```diff
@@ -99,3 +117,35 @@ def url_parsed(self):
     @property
     def query_params(self):
         return self.url_parsed.query
+
+
+class OutgoingRequestsLogConfig(SingletonModel):
+    class SaveLogsChoices(models.IntegerChoices):
+        NO = 0, _("No")
+        YES = 1, _("Yes")
+
+        __empty__ = _("Use default")
+
+    save_to_db = models.IntegerField(
+        _("Save logs to database"),
+        choices=SaveLogsChoices.choices,
+        blank=True,
+        null=True,
+        help_text=_(
+            "Whether request logs should be saved to the database (default: {default})"
+        ).format(default=settings.LOG_OUTGOING_REQUESTS_DB_SAVE),
```

Comment:
you probably want to map this to get consisten labels:

```py
mapping = {
    True: SaveLogsChoices.yes,
    False: SaveLogsChoices.no,
}

# and then in the help text
(...).format(default = mapping[settings.LOG_OUTGOING_REQUESTS_DB_SAVE].label)
```

HOWEVER - this dynamic help text should not be specified at the model level, but at the admin or form level. If you use different values for the setting than the default of the library, this will result in a different help text string and be picked up by Django's `makemigrations`, so you'll end up with migrations that are not in the library and will pollute your virtual env and will even cause migration history order problems.

I'd define the form:

```py
class OutgoingRequestsLogConfigForm(forms.ModelForm):
    class Meta:
        model = OutgoingRequestsLogConfig
        fields = "__all__"
        help_texts = {
            "save_to_db": _("...").format(default=...),  # dynamic help text based on value of settings.
        }
```

and then in the admin you can use this:

```py
class OutgoingRequestsLogConfigAdmin(SoloAdmin):
    form = OutgoingRequestsLogConfigForm
```